### PR TITLE
fix: fractional operator name

### PR DESCRIPTION
**Pull request by toddbaert**
* fixes `fractionalEvaluation` name (should be `fractional`)
* adds e2e suite for custom operators in in-process provider

### CODE REVIEW

File: providers/flagd-in-process/pkg/provider.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func NewProvider(ctx context.Context, opts ...ProviderOption) *Provider {
 		provider.logger,
 		s,
 		eval.WithEvaluator(
-			"fractionalEvaluation",
+			"fractional",
```
Comment (toddbaert) [inline providers/flagd-in-process/pkg/provider.go:130]: This is the only functional change.

**Review by beeme1mr — APPROVED** [review]

File: .github/component_owners.yml [hunk 2]
```diff
@@ -10,6 +10,10 @@ components:
   providers/flagd:
     - Kavindu-Dodan
     - toddbaert
+  providers/flagd-in-process:
+    - bacherfl
+    - Kavindu-Dodan
+    - toddbaert
```
Comment (Kavindu-Dodan) [inline .github/component_owners.yml:16]: 🎉

File: providers/flagd-in-process/pkg/provider.go (hunk 1 above)
Comment (Kavindu-Dodan) [reply providers/flagd-in-process/pkg/provider.go:130]: Will this be a breaking change 🤔  ?

**Review by Kavindu-Dodan — APPROVED** [review]

File: providers/flagd-in-process/pkg/provider.go (hunk 1 above)
Comment (toddbaert) [reply providers/flagd-in-process/pkg/provider.go:130]: sort-of... but it was a bug in the first place. As per spec/doc the operator was always supposed to be called `fractional`.

Also we haven't released it yet :smile:

**Comment by toddbaert** [comment]
@beeme1mr @Kavindu-Dodan note, I've added this package to to release please config. I think we can release after this.

File: providers/flagd-in-process/pkg/provider.go (hunk 1 above)
Comment (Kavindu-Dodan) [reply providers/flagd-in-process/pkg/provider.go:130]: Ahh yeah, sorry I forgot the fact that the Go in-process provider is a separate package :)

**Pull request merged**